src/legacy/core_plugins/kibana/public/management/sections/settings/advanced_settings.scss

```diff
@@ -5,9 +5,25 @@
 
   &Wrapper {
     width: 640px;
+
+    @include internetExplorerOnly() {
+      min-height: 1px;
+    }
   }
 
   &Actions {
     padding-top: $euiSizeM;
   }
+
+  &ImagePreview > img {
```

Comment:
I would just err on not overriding EUI, and this will be updated automatically when that issue is addressed. It's quite an edge case that I don't think we need to worrry about at the moment.

Reply:
Totally agree, I removed it for now. We have the issue in EUI to track that.